.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Unit Tests
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  unit-test:
+    fail-fast: false
```

Comment:
I think this line is supposed to be in the `stragegy` section

Reply:
Slight slip (didn't notice since test passed). Fixed

Reply:
Test didn't actually run

Reply:
I might have added it last second then 😅 It should be working now.